rllib/core/models/configs.py

```diff
@@ -229,20 +303,16 @@ class FreeLogStdMLPHeadConfig(_MLPConfig):
     """
 
     def _validate(self, framework: str = "torch"):
-        actual_output_dims = (
-            [self.hidden_layer_dims[-1]]
-            if self.output_dims is None
-            else self.output_dims
-        )
-        if len(actual_output_dims) > 1 or actual_output_dims[0] % 2 == 1:
+        if len(self.output_dims) > 1 or self.output_dims[0] % 2 == 1:
```

Comment:
self.output_dims[-1]?

Reply:
True, but it should have len=1 anyways, no? Correct me if I'm wrong.

Reply:
Yeah, I mean it doesn't matter. The if-statement already checks that len is 1 (otherwise, this would yield the same error).